bootstrap: Panic comparing errors

https://ci.flynn.io/builds/20151127182442-81b8d73e

```
++ 18:30:23.021 bootstrap ===> postgres start
panic: runtime error: comparing uncomparable type httphelper.JSONError [recovered]
        panic: runtime error: comparing uncomparable type httphelper.JSONError

goroutine 1 [running]:
github.com/flynn/flynn/pkg/shutdown.(*handler).exit(0xc208088d20, 0x0, 0x0, 0x0, 0xa7dba0, 0xc2081314c0)
        /home/ubuntu/go/src/github.com/flynn/flynn/pkg/shutdown/handler.go:69 +0x12e
github.com/flynn/flynn/pkg/shutdown.Exit()
        /home/ubuntu/go/src/github.com/flynn/flynn/pkg/shutdown/handler.go:40 +0x73
github.com/flynn/flynn/host/cli.runBootstrap(0xc20801ec20, 0x0, 0x0)
        /home/ubuntu/go/src/github.com/flynn/flynn/host/cli/bootstrap.go:109 +0x957
github.com/flynn/flynn/host/cli.Run(0x7ffd91ecad78, 0x9, 0xc208039dc0, 0x4, 0x4, 0x0, 0x0)
        /home/ubuntu/go/src/github.com/flynn/flynn/host/cli/cli.go:62 +0x677
main.main()
        /home/ubuntu/go/src/github.com/flynn/flynn/host/host.go:127 +0x8c1
...
++ 18:30:23.447 bootstrap ===> postgres error
++ 18:30:23.448 : 
```

Likely introduced by the change in #2170.

--- a/host/cli/bootstrap.go
+++ b/host/cli/bootstrap.go
@@ -106,7 +106,7 @@ func runBootstrap(args *docopt.Args) error {
 	}
 
 	<-done
-	if err != nil && err == last {
+	if err != nil && last != nil && err.Error() == last.Error() {
 		return ErrAlreadyLogged{err}
 	}
 	return err

